dev_guide/deployments.adoc

```diff
@@ -145,51 +166,46 @@ before the cancellation is effective.
 To retry the last failed deployment:
 
 ----
-$ oc deploy <deployment_config> --retry
+$ oc deploy --retry dc/<name>
 ----
 
 If the last deployment didn't fail, the command will display a message and the
 deployment will not be retried.
 
 NOTE: Retrying a deployment restarts the deployment and does not create a new
-deployment version. The restarted deployment will have the same configuration
+deployment revision. The restarted deployment will have the same configuration
 it had when it failed.
 
 [[rolling-back-a-deployment]]
 == Rolling Back a Deployment
 
-Rollbacks revert an application back to a previous deployment and can be
+Rollbacks revert an application back to a previous template and can be
 performed using the REST API, the CLI, or the web console.
 
-To rollback to the last successful deployment:
+To rollback to the last successful deployed revision of your configuration:
 
 ----
-$ oc rollback <deployment_config>
+$ oc rollout undo dc/<name>
 ----
 
-The deployment configuration's template will be reverted to match the
-deployment specified in the rollback command, and a new deployment will be
-started.
+The deployment configuration's template will be reverted to match the deployment
+revision specified in the undo command, and a new replication controller will be started.
+If no revision is specified with --to-revision, then the last successfully deployed
+revision will be used.
 
 Image change triggers on the deployment configuration are disabled as part of
 the rollback to prevent unwanted deployments soon after the rollback is
 complete. To re-enable the image change triggers:
 
 ----
-$ oc deploy <deployment_config> --enable-triggers
+$ oc deploy --enable-triggers dc/<name>
```

Comment:
@kargakis shouldn't we promote the `oc set triggers --from-image=foo dc/<name>` here?